stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -8,12 +8,14 @@ public data class Config(
     /**
      * Date of channel creation.
      */
+    @Deprecated("User createdAt instead", level = DeprecationLevel.WARNING, replaceWith = ReplaceWith("createdAt"))
```

Comment:
I hate to leave another comment on this PR because it must feel like nitpicking, but for some reason this says "User" instead of "User" 🙈

```suggestion
    @Deprecated("Use createdAt instead", level = DeprecationLevel.WARNING, replaceWith = ReplaceWith("createdAt"))
```